src/assets/courses/full-courses.json

```diff
@@ -690,7 +694,7 @@
         }
       ],
       "catalogWhenOffered": "Spring.",
-      "catalogComments": " Live animals used in course instruction.",
+      "catalogComments": " Live animals used in course instruction.",
```

Comment:
what's with the change in space? I thought #720 would've fixed this.

Reply:
The replacement should have been a `.replaceAll()` instead of a `.replace()`. Can we hold off on fixing this until the next release?

Reply:
Maybe it should've been a replaceAll, but that doesn't explain why we're having a different result now when (I assume) the API returned the same data.

Reply:
Are you sure you ran it while branched off of master? After #727, there were still 861 Grelim spaces. After this PR, all the Grelim spaces were replaced.

Reply:
I probably didn't run it off of master 🤦🏼‍♂️. I waited to run it until I had fixed the script, and I thought I did run it with the changes because I think I didn't have any `null` issues when I reran it that initially caused me to make #720. Maybe Cornell removed the nulls from their data, which meant I didn't catch that the script wasn't updated. #729 also didn't catch it because that was only the req-gen.